apollo-espresso-support/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Not sure if the top level .gitignore ignores the module specific build directories, so yeah this is indeed required.